content/firstabout.html

```diff
@@ -1,24 +1,24 @@
-<a href="http://www.usfirst.org">FIRST (For Inspiration and Recognition of Science and Technology)</a> is a world-wide organization started
-by Dean Kamen. It was designed to encourage students to pursue careers in science and technology,
-and to instill a sense of Gracious Professionalism. Gracious Professionalism encourages students to work
-together to help other teams when needed and to treat everyone with kindness and respect. Because of
-this, FIRST is said to not only teach students to build robots, but to build character as well.
+<a href="http://www.usfirst.org">FIRST (For Inspiration and Recognition of Science and Technology)</a> is a world-wide organization founded 
+by Dean Kamen in 1992. FISRT was designed to encourage students to pursue S.T.E.M. related careers 
+while instilling a sense of Gracious Professionalism. Gracious Professionalism is the mutual respect of oneself and others within 
+the heat of competition. FIRST is built around four levels of robotics competition. <a href="http://www.usfirst.org/roboticsprograms/jr.fll">Jr.FLL(grades K-3)</a>, 
+<a href="http://www.usfirst.org/roboticsprograms/fll">FLL(4-8)</a>, 
+<a href="http://www.usfirst.org/roboticsprograms/ftc">FTC(7-12)</a>, and 
+<a href="http://www.usfirst.org/roboticsprograms/frc">FRC(9-12)</a> enable students of all ages to 
+engage in FIRST.
 
 <br><br>
 <img src="images/AboutFIRST1.JPG">
 <br><br>
 
-The FIRST Robotics Competition (FRC) which our team competes in is &quot;the varsity sport of the mind.&quot;
-FRC students have six weeks to design, prototype and build fully functioning competition robots that
-can perform a certain task, which changes every year. In addition to having a robot, it is important and
-is encouraged for teams to develop a strong sense of themselves and to market themselves and their
-robots at competitions.
+<a href="http://www.usfirst.org/roboticsprograms/frc">The FIRST Robotics Competition(FRC)</a>, which our team competes in, is the &quot;varsity sport of the mind&quot;. 
+FRC students have six weeks to design, prototype and build fully functioning competition robots that 
+can perform efficently in a challenge based game. In addition to building a robot, teams must market 
+and communicate themselves effectively to one another in order to develop resilient alliances.
 
 <br><br>
 <img src="images/AboutFIRST2.JPG">
 <br><br>
 
-The FRC program reaches almost 60 thousand students across 12 different countries including Chile,
-Australia and Turkey. Each season these students compete against one another at various regional
-competitions across the United States and in their own countries. Each team is competing for a spot at the
-World Championship which is held in St. Louis, Missouri every April.
+Every year, FRC reaches over 78,000 students across 130 different competitions. These students make up the nearly
+3,100 teams that will compete for one of the 800 spots at the World Championships in St. Louis, Missouri.
```

Comment:
I think it is still worth pointing out that FIRST is an international program, which was implied by mentioning 12 different countries.